src/plugins/data/public/query/query_string/query_string_manager.mock.ts

```diff
@@ -44,10 +45,10 @@ const createSetupContractMock = (isEnhancementsEnabled: boolean = false) => {
   };
 
   const queryStringManagerMock: jest.Mocked<QueryStringContract> = {
-    getQuery: jest.fn(),
+    getQuery: jest.fn().mockReturnValue(defaultQuery),
     setQuery: jest.fn(),
-    getUpdates$: jest.fn(),
-    getDefaultQuery: jest.fn(),
+    getUpdates$: jest.fn().mockReturnValue(of(defaultQuery)),
```

Comment:
nice.

but i think one of the tests will fail now. so might have to just update the test to expect this now.

Reply:
weird i ran `yarn test:jest` before pushing, they passed locally